src/firebase/auth.js

```diff
@@ -0,0 +1,48 @@
+import { auth } from "./firebase";
+import {
+  createUserWithEmailAndPassword,
+  GoogleAuthProvider,
+  sendEmailVerification,
+  sendPasswordResetEmail,
+  signInWithEmailAndPassword,
+  signInWithPopup,
+  updatePassword,
+} from "firebase/auth";
+
+export const doCreateUserWithEmailAndPassword = async (email, password) => {
+  return createUserWithEmailAndPassword(auth, email, password);
+};
```

Comment:
Can we change the function names in auth domain: remove "do" in the beginning and add "auth" prefix? 
```suggestion
export const authCreateUser = async (email, password) => {
  return createUserWithEmailAndPassword(auth, email, password);
};
```